miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/RealMiniAppDisplay.kt

```diff
@@ -58,11 +63,21 @@ internal class RealMiniAppDisplay(
     // not rendered successfully in the mini app e.g. select tags
     override suspend fun getMiniAppView(activityContext: Context): View? =
         if (isContextValid(activityContext)) {
+            getMiniAppAnalytics()?.sendAnalytics(
```

Comment:
I think, the good place to send this one inside `provideMiniAppWebView` after ensuring that the `MiniAppWebView` is displaying?

Reply:
It's tracking the behavior that developers open mini app view, I guess we don't track whether `MiniAppWebView` init properly.